testing/content/files/en-us/web/images/bad_src/index.html

```diff
@@ -0,0 +1,16 @@
+---
+title: <img> tag with src that isn't a valid image
+slug: Web/Images/Bad_src
+---
+
+<!-- exists on disk but isn't actually a real SVG file. -->
+<img src="actuallynota.svg">
+
+<!-- resolves, intially, to a directory -->
+<img src="/en-us/xxx">
```

Comment:
@peterbe `testing/content/en-us/xxx` doesn't exist does it? Should we replace this with
```suggestion
<img src="/en-us/web">
```

Reply:
No, what happens is that it thinks the `xxx` is the image name and that the parent folder is `/en-us/` which it then proceeds to try to look up. And yes, that does exist (`/path/to/content/files/en-us`) so it thinks it's found the image.

Reply:
If I read the code correct it thinks `xxx` is the `docs` part in a URL and ignores it (see https://github.com/mdn/yari/pull/3191/files#diff-5883f304c7715ab1e699e66f97127621d440a6ca7bbb7d8989e6978066b2fd60R34)

Can we change it to some thing more obvious like `/en-US/docs/Web` so it's clear that it points to a folder?